highCard.rb

```diff
@@ -0,0 +1,29 @@
+
+puts "Player 1, what is your name? #{input = @playerOneName}"
+puts "Player 2, what is your name? #{input = @playerTwoName}"
+
+suit = [clubs, diamonds, hearts, spades]
+
+rank = rank.new
+%w(ace king queen jack).each_with_index {|item, index|
+  rank[item] = index
+}
+
+@jack = 11
```

Comment:
Not entirely sure why you used `@` here... You only use that when you're inside a `class`, as in:

```rb
class Person

end
```